Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml

```diff
@@ -27,7 +27,7 @@
 
             <TextView
                 android:id="@+id/txtDeviceLabel"
-                android:layout_width="70dp"
+                android:layout_width="100dp"

```

Comment:
why 100? is it better?

Reply:
Russian text no longer fits and is being cut badly. I suspect other translations wont fit either.

Reply:
got it!  thanks